node/commons/compatibility-layer.ts

```diff
@@ -262,13 +261,9 @@ const convertSKU = (
   }
 
   variations.forEach((variation) => {
-    const attributes = product.textAttributes.filter((attribute) => attribute.labelKey == variation)
-    if (attributes != null && attributes.length > 0) {
-      item[variation] = []
-
-      attributes.forEach((attribute) => {
-        item[variation].push(attribute.labelValue)
-      })
+    const attribute = sku.attributes.find((attribute) => attribute.key == variation)
+    if (attribute != null) {
+      item[variation] = [attribute.value]
```

Comment:
@LucasTornai this one I need your attention, this is the only difference between this PR and the https://github.com/vtex-apps/search-resolver/pull/74, as this caused polishop products to not have variation, because we're sending variations from the SKU, but getting the attribute information from the product, causing in some cases the value to be `undefined`.

Reply:
That doesn't make a lot of sense to me. Technically, all the SKUs attributes should also be in the product TextAttributes, else, how could you filter by those attributes? Yet, I checked polishop, and it do happen there, some attributes in the SKUs are not appearing in the TextAttributes.